packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -163,19 +160,17 @@ export const useProductFullDetail = props => {
         productType
     );
 
-    const [{ isAddingItem }, { addItemToCart }] = useCartContext();
+    const [{ cartId }] = useCartContext();
 
-    const [addConfigurableProductToCart] = useMutation(
-        addConfigurableProductToCartMutation
-    );
-
-    const [addSimpleProductToCart] = useMutation(
-        addSimpleProductToCartMutation
-    );
-
-    const [fetchCartId] = useMutation(createCartMutation);
+    const [
+        addConfigurableProductToCart,
+        { error: addConfigurableError, loading: addConfigurableLoading }
+    ] = useMutation(addConfigurableProductToCartMutation);
 
-    const fetchCartDetails = useAwaitQuery(getCartDetailsQuery);
+    const [
+        addSimpleProductToCart,
+        { error: addSimpleError, loading: addSimpleLoading }
+    ] = useMutation(addSimpleProductToCartMutation);
```

Comment:
We need to add `MiniCartFragment` to these mutation response bodies depending on whether #2506 is merged first or this one. Please be aware of that @revanth0212 @dpatil-magento